dev/dune

```diff
@@ -0,0 +1,14 @@
+(library
+ (name debug_printers)
+ (public_name coq.debug_printers)
+ (synopsis "Coq's Debug Printers")
+ (modules :standard \ checker_printers)
+ (libraries coq.plugins.ltac))
+
+(library
+ (name checker_printers)
+ (public_name coq.checker_printers)
+ (synopsis "Coq's Debug Printers [for the Checker]")
+ (flags :standard -open Checklib)
+ (modules checker_printers)
+ (libraries coq.checklib))
```

Comment:
Why's this included?

Reply:
No special reason other than making my own dev setup work properly.